src/20241106_Multi_GHOAvaxLaunch/AaveV3Arbitrum_GHOAvaxLaunch_20241106.t.sol

```diff
@@ -0,0 +1,500 @@
+  address public constant AVAX_RMN_PROXY = 0xcBD48A8eB077381c3c4Eb36b402d7283aB2b11Bc;
+  address public constant AVAX_ROUTER = 0xF4c7E640EdA248ef95972845a62bdC74237805dB;
+  address public constant ETH_TOKEN_POOL = MiscEthereum.GHO_CCIP_TOKEN_POOL;
+  address public constant CCIP_ARB_ETH_ON_RAMP = 0x67761742ac8A21Ec4D76CA18cbd701e5A6F3Bef3;
+  address public constant CCIP_ARB_ETH_OFF_RAMP = 0x91e46cc5590A4B9182e47f40006140A7077Dec31;
+  address public constant CCIP_ARB_AVAX_ON_RAMP = 0xe80cC83B895ada027b722b78949b296Bd1fC5639;
+  address public constant CCIP_ARB_AVAX_OFF_RAMP = 0x95095007d5Cc3E7517A1A03c9e228adA5D0bc376;
+  address public constant TOKEN_POOL_AND_PROXY = 0x26329558f08cbb40d6a4CCA0E0C67b29D64A8c50;
+  uint64 public constant CCIP_AVAX_CHAIN_SELECTOR = 6433500567565415381;
+
+  event Minted(address indexed sender, address indexed recipient, uint256 amount);
+  event Burned(address indexed sender, uint256 amount);
+  event Transfer(address indexed from, address indexed to, uint256 value);
+
+  function setUp() public {
+    // Execute Avax proposal to deploy Avax token pool
+    vm.createSelectFork(vm.rpcUrl('avalanche'), 53559217);
+
+    // TODO: Decide if we want to deploy this beforehand or in AIP
+    _deployGhoToken();
+
+    // TODO: Remove this deployment once we have deployed pool address
+    _deployCcipTokenPool();
+
+    // TODO: Remove this (will be done on chainlink's side)
+    // Prank chainlink and set up admin role to be accepted on token registry
+    vm.startPrank(AVAX_REGISTRY_ADMIN);
+    TokenAdminRegistry(AVAX_TOKEN_ADMIN_REGISTRY).proposeAdministrator(
+      AVAX_GHO_TOKEN,
+      GovernanceV3Avalanche.EXECUTOR_LVL_1
+    );
+    vm.stopPrank();
+
+    AaveV3Avalanche_GHOAvaxLaunch_20241106 avaxProposal = new AaveV3Avalanche_GHOAvaxLaunch_20241106();
+    GovV3Helpers.executePayload(vm, address(avaxProposal));
+
+    // Switch to Arbitrum and create proposal
+    vm.createSelectFork(vm.rpcUrl('arbitrum'), 279521658);
+
+    // Configure TokenPoolAndProxy for Avalanche
+    // Prank Registry owner
+    vm.startPrank(REGISTRY_ADMIN);
+    _configureCcipTokenPool(TOKEN_POOL_AND_PROXY, CCIP_AVAX_CHAIN_SELECTOR);
+    vm.stopPrank();
+    proposal = new AaveV3Arbitrum_GHOAvaxLaunch_20241106();
+  }
+
+  /**
+   * @dev executes the generic test suite including e2e and config snapshots
+   */
+  function test_defaultProposalExecution() public {
+    defaultTest('AaveV3Arbitrum_GHOAvaxLaunch_20241106', AaveV3Arbitrum.POOL, address(proposal));
+
+    _validateCcipTokenPool();
+  }
+
+  /// @dev Test burn and mint actions, mocking CCIP calls
+  function test_ccipTokenPool() public {
+    GovV3Helpers.executePayload(vm, address(proposal));
+
+    // Mock calls
+    address router = TOKEN_POOL.getRouter();
+    address ramp = makeAddr('ramp');
+    vm.mockCall(
+      router,
+      abi.encodeWithSelector(bytes4(keccak256('getOnRamp(uint64)'))),
```

Comment:
you dont need to mockCall on/offRamps on arb since they already exist, ideally i think these tests should re-use the on chain on/offRamps and router, add the avax token pool to TokenAdminRegistry; and simply check ccipSend, onRamp (lockOrBurn), works when sending ARB -> AVAX and back for offRamp (releaseOrMint)  
we would need to probably do this for avax-chain specific tests

Reply:
actually idt you need to do this for avax as well since AVAX<>ETH & AVAX<>ARB lanes are already configured https://docs.chain.link/ccip/directory/mainnet/chain/avalanche-mainnet
will be interesting to add tests for the different fee-tokens